doc/classes/SplitContainer.xml

```diff
@@ -33,6 +59,16 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="drag_released">
+			<description>
+				Emitted when the mouse is released from the drag area.
```

Comment:
```suggestion
		<signal name="drag_finished">
			<description>
				Emitted when the user finished dragging.
```

`finished` matches `started` better. Also the mouse won't necessarily be in the drag area.

Reply:
The signals are emitted on mouse press/release, rather than arming a bool then emitting on mouse movement, which I think adds unnecessary steps for essentially equivalent behavior.

How about we rename the signals to `dragging_pressed` and `dragging_released` for consistency?

Reply:
I was thinking that in case the user is using a touch screen or pen tablet, we shouldn't mention the mouse.
I agree that the current behavior is fine, but I don't see how `drag_finished` implies changing it? We can change the description if it's misleading.

Reply:
I see your point - I'll use `drag_finished` and adjust the description accordingly. Thanks!

Reply:
`drag_ended` is used by `ScrollContainer` and `Range`, so how about we use that?

Reply:
`drag_ended` works too.